tools/changelog-helper.sh

```diff
@@ -126,7 +126,7 @@ check_or_add_pr() {
         return
     fi
     checked_ids[$id]=1
-    if grep -qF "#$id" <<<"$changelog"; then
+    if grep -qP "#$id([^0-9]|$)" <<<"$changelog"; then
```

Comment:
Doesn't `"#$id\>"` work?  I don't think I'd want it matching `#12AB`, either.

Reply:
> Doesn't `"#$id\>"` work?

I didn't know about that -- thanks! It works in `-E` mode (may `\b` might have worked there, too). Updated.

Reply:
I think `\b` will only work in `-P` (PCRE) mode, if the installed version of `grep` supports it. I prefer `\<` and `\>` if I'm not actually writing perl or PHP.